src/diagnostics/define_diagnostics.jl

```diff
@@ -179,130 +181,276 @@ function define_diagnostics!(land_model)
         compute! = (out, Y, p, t) ->
             compute_respiration_leaf!(out, Y, p, t, land_model),
     )
 
+    # Vcmax25
     add_diagnostic_variable!(
         short_name = "vcmax25",
         long_name = "Vcmax25",
         standard_name = "vcmax25",
-        units = "",
+        units = "mol CO2 m^-2 s^-1",
         comments = "The parameter vcmax at 25 degree celsius. Important for the Farquhar model of leaf photosynthesis.",
         compute! = (out, Y, p, t) -> compute_vcmax25!(out, Y, p, t, land_model),
     )
 
+    ### Canopy - Radiative Transfer
+    # NIR - near infrared radiaton
+    add_diagnostic_variable!(
+        short_name = "nir",
+        long_name = "Near Infrared Radiation",
+        standard_name = "near_infrared_radiation",
+        units = "mol photons m^-2 s^-1",
+        comments = "The amount of near infrared radiation reaching the canopy.",
+        compute! = (out, Y, p, t) ->
+            compute_near_infrared_radiation!(out, Y, p, t, land_model),
+    )
+
+    # ANIR - absorbed near infrared radiation
+    add_diagnostic_variable!(
+        short_name = "anir",
+        long_name = "Absorbed Near Infrared Radiation",
+        standard_name = "absorbed_near_infrared_radiation",
+        units = "mol photons m^-2 s^-1",
+        comments = "The amount of near infrared radiation absorbed by the canopy.",
+        compute! = (out, Y, p, t) ->
+            compute_near_infrared_radiation_absorbed!(out, Y, p, t, land_model),
+    )
+
+    # RNIR - reflected near infrared radiation
+    add_diagnostic_variable!(
+        short_name = "rnir",
+        long_name = "Reflected Near Infrared Radiation",
+        standard_name = "reflected_near_infrared_radiation",
+        units = "mol photons m^-2 s^-1",
+        comments = "The amount of near infrared radiation reflected by the canopy.",
+        compute! = (out, Y, p, t) ->
+            compute_near_infrared_radiation_reflected!(
+                out,
+                Y,
+                p,
+                t,
+                land_model,
+            ),
+    )
+
+    # TNIR - transmitted near infrared radiation
+    add_diagnostic_variable!(
+        short_name = "tnir",
+        long_name = "Transmitted Near Infrared Radiation",
+        standard_name = "transmitted_near_infrared_radiation",
+        units = "mol photons m^-2 s^-1",
+        comments = "The amount of near infrared radiation transmitted by the canopy.",
+        compute! = (out, Y, p, t) ->
+            compute_near_infrared_radiation_transmitted!(
+                out,
+                Y,
+                p,
+                t,
+                land_model,
+            ),
+    )
+
+    # PAR - photosynthetically active radiation
     add_diagnostic_variable!(
         short_name = "par",
         long_name = "Photosynthetically Active Radiation",
         standard_name = "photosynthetically_active_radiation",
-        units = "",
+        units = "mol photons m^-2 s^-1",
```

Comment:
For diagnostics, energy is usually in W.m-2. Much easier to compare with observations.

Reply:
@kmdeck 
Currently we are computing PAR and NIR and others in mol photons m-2 s-1. 
See for example https://github.com/CliMA/ClimaLand.jl/blob/main/src/standalone/Vegetation/canopy_parameterizations.jl#L623

We could convert it to W m-2. Do you think we should? 
(it's easy to convert in data analysis, and we are probably going to convert most units anyway because we use SI)

Reply:
I think some photosynthesis computation required APAR in moles of photons, right? 

For now I would leave source code the same, but we can convert units in diagnostics?